dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/SchedulerService.java

```diff
@@ -128,12 +128,11 @@ Schedule getSchedule(User loginUser,
      * @param projectCode project code
      * @param id scheduler id
      * @param scheduleStatus schedule status
-     * @return publish result code
      */
-    Map<String, Object> setScheduleState(User loginUser,
-                                         long projectCode,
-                                         Integer id,
-                                         ReleaseState scheduleStatus);
+    void setScheduleState(User loginUser,
```

Comment:
Just curious, why to make setScheduleState no more return value?

Reply:
In the previous, we just return the error message inside the hashmap, we should avoid wrapper error but directly throw the exceptions in the code.

Reply:
Using exception wrapper in hashmap will make the code too long and hard to test, you can see the previous `testSetScheduleState` in `SchedulerServiceTest.java` never test actual success